ui/src/environments/commonEnvironment.ts

```diff
@@ -0,0 +1,21 @@
+const contentUrlsDe = {
```

Comment:
Why do we need the angular environment feature here? These values don't need to change at build time, right?

Reply:
Done